lib/src/google_sign_in_mocks_base.dart

```diff
@@ -18,6 +18,13 @@ class MockGoogleSignInAccount extends Mock implements GoogleSignInAccount {
   @override
   Future<GoogleSignInAuthentication> get authentication =>
       Future.value(MockGoogleSignInAuthentication());
+
+  @override
+  bool operator ==(other) {
```

Comment:
Do we need this? It seems like this is the exact same code as Mock's == code: https://github.com/dart-lang/mockito/blob/171d4d60f59826c0c1c84ddb98b6161474a80bea/lib/src/mock.dart#L195

I'd rather we didn't add in this PR it if it is not needed for non-nullability.